.github/workflows/release.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ${{ matrix.os }}
```

Comment:
In zap-schema.svg file I see the following:

<g id="node16" class="node">

Reply:
thanks, will fix